src/view.c

```diff
@@ -267,6 +267,34 @@ view_minimize(struct view *view, bool minimized)
 	}
 }
 
+static void
+minimize_sub_views(struct view *view, bool minimized)
+{
+	struct view **child;
+	struct wl_array children = { 0 };
+
+	view_get_children(view, &children);
```

Comment:
I am not really familiar with wl_array, no `wl_array_init()` required here?

Reply:
No because we set to zero with `= { 0 }`. All `wl_array_init()` does is a memset to zero.

https://gitlab.freedesktop.org/wayland/wayland/-/blob/main/src/wayland-util.c#L96-100

Reply:
Interesting.
So technically its not necessary at the moment.

I'd still argue that we should always use it to initialize a wl_array as the way it works is an implementation detail of wl_array. They may change it in the future, (e.g. they could add an initial `malloc()` in there and initialize some counters or similar while keeping the API the same).

Reply:
Okay. Have reverted to using `wl_array_init()`.